test/python/sdpa/frost/test_sdpa_fwd_fp8_sm120.py

```
@@ -0,0 +1,250 @@
+    if sdpa_kwargs.get("use_causal_mask"):
+        out["is_causal"] = True
+    if sdpa_kwargs.get("use_causal_mask_bottom_right"):
+        out["is_causal"] = True
+        out["bottom_right"] = True
+    lb = sdpa_kwargs.get("left_bound")
+    if lb is not None:
+        out["swa_window"] = lb - 1
+    return out
+
+
+def _check(O, O_ref, amax_s, amax_s_ref, amax_o, amax_o_ref):
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)